drive-stack/conUDS/defs.bzl

```diff
@@ -3,19 +3,31 @@ load("@prelude//:rules.bzl", __rules__ = "rules")
 def conUDS_download(
         name: str,
         binary: str,
-        node: str):
+        node: str,
+        manifest: str | None = None):
+    args = ["-m"]
+    if manifest:
+        args.append("$(location {})".format(manifest))
+    else:
+        args.append("$(location //network:manifest-uds)")
```

Comment:
hmmmm this seems weird, what are you trying to do here?

Reply:
Default to the generic network but this is ret lol. putting it to require the user to provide a manifest